src/isIsogram.test.js

```diff
@@ -3,7 +3,39 @@
 describe('isIsogram', () => {
   const { isIsogram } = require('./isIsogram');
 
-  it(`should be declared`, () => {
+  it('should be declared as a function', () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for a word with no repeating letters', () => {
+    expect(isIsogram('playgrounds')).toBe(true); // все буквы уникальны
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('look')).toBe(false); // буква "o" повторяется
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('Adam')).toBe(false);
+  });
```

Comment:
The description for this test case is not unique. Consider changing it to something like 'should return false for a word with repeating letters regardless of case' to better reflect the case-insensitivity requirement from the task description.

Reply:
Needs a fix here.